galaxy_ng/app/dynaconf_hooks.py

```diff
@@ -533,6 +534,16 @@ def configure_authentication_backends(settings: Dynaconf) -> Dict[str, Any]:
     return data
 
 
+def configure_renderers() -> Dict[str, Any]:
+    "Set default renderer classes."
+    return {
+        "REST_FRAMEWORK__DEFAULT_RENDERER_CLASSES": [
+            'rest_framework.renderers.JSONRenderer',
+            'galaxy_ng.app.renderers.CustomBrowsableAPIRenderer'
+        ]
+    }
```

Comment:
There is no need to use dynaconf_hooks to set default values without any conditional, the hooks are valuable when the return dict to be merged with settings can vary depending on a condition.

The reason to perform the conditional here is that the hooks file runs at the very late stage of settings building, if the conditional is performed inside the `settings.py` the state of `settings` objects is unfinished, for example, another plugin can set a variable to override it.

```suggestion
import re

def configure_renderers(settings) -> Dict[str, Any]:
    "Set renderer classes conditionaly."
    if re.search(
        r'galaxy(-dev|-stage)*.ansible.com', settings.get('CONTENT_ORIGIN', "")
    ):
        # overrride the `REST_FRAMEWORK__DEFAULT_RENDERER_CLASSES`
        return {
            "REST_FRAMEWORK__DEFAULT_RENDERER_CLASSES": [
                'galaxy_ng.app.renderers.CustomBrowsableAPIRenderer'
            ]
        }
        
    return {}   # do nothing
```